py_ecc/optimized_bls12_381/constants.py

```diff
@@ -0,0 +1,49 @@
+from py_ecc.fields import (
+    optimized_bls12_381_FQ2 as FQ2,
+)
+
+# Ciphersuite BLS12381G2-SHA256-SSWU-RO paramters
+ISO_3_A = FQ2([0, 240])
+ISO_3_B = FQ2([1012, 1012])
+ISO_3_Z = FQ2([1, 1])
+P_MINUS_9_DIV_16 = 1001205140483106588246484290269935788605945006208159541241399033561623546780709821462541004956387089373434649096260670658193992783731681621012512651314777238193313314641988297376025498093520728838658813979860931248214124593092835  # noqa: E501
+SQRT_I = 1028732146235106349975324479215795277384839936929757896155643118032610843298655225875571310552543014690878354869257  # noqa: E501
```

Comment:
I have some discomfort from having all of these constants floating around.  Is there a formal/canonical spec that we can include a link to so that it's easy to reference/cross-check these constants are indeed the values they are supposed to be?

Reply:
There are soo many constants! Unfortunately they are necessary for the isogeny mapping.

Most of them come from
BLS General Constants: https://tools.ietf.org/html/draft-irtf-cfrg-hash-to-curve-04#section-8.7
ISO constants: https://tools.ietf.org/html/draft-irtf-cfrg-hash-to-curve-04#appendix-C.2